include/bl_storage.rst

```diff
@@ -0,0 +1,32 @@
+.. _doc_bl_storage:
+
+Bootloader storage
+##################
+
+The bootloader storage library is used by the :ref:`bootloader` to read from and write to a protected data area that contains the provisioned bootloader data.
+The library can also be included into the application to allow reading the provisioned data.
+
+The provisioned data is stored in the one-time programmable (OTP) region of UICR, if available (for example, on nRF9160 and the nRF5340 application core).
+In this case, it can only be read from applications that are in the secure domain.
+If OTP is not available, the data is stored in a regular flash page, and the bootloader blocks write access to it before booting the next image.
+In this case, some functions do not work when they are called from the application.
+
+The library stores the following data:
+
+* Image slot sizes
+* Public keys
```

Comment:
Should we say here that we store the hash?

Reply:
Change to "Root of trust public key hashes"? Or "Hashes of public keys"?

Reply:
Changed to "Hashes of public keys".